packages/common/src/api-dtos.ts

```diff
@@ -63,6 +63,14 @@ export class UpdatePlanDto {
 }
 
 export class SignUpStudentDto {
+  @IsOptional()
```

Comment:
Do we want to include this here or is this just to fit the current designs and will be later removed in a future PR?

Reply:
didn't realize the student entity had a fullname and not a first and last name... i'll update this to be an optional full name